hackathon_site/dashboard/frontend/src/components/general/PopupModal/PopupModal.module.scss

```diff
@@ -0,0 +1,32 @@
+@import "../../../assets/abstracts/mixins";
+@import "../../../assets/abstracts/variables";
+
+.modalSize {
+    position: absolute;
+    //margin: auto;
+    margin-top: 20px;
+    top: 40%;
```

Comment:
For centering, we usually use flexbox because it prevents hardcoding. Can you remove all the margin, positioning and transform styles from .modalSize

Instead, create a new style to put directly in the Modal component with these styles:
```
            display: 'flex',
            align-items: 'center',
            justify-content: 'center',
```

Reply:
done